<!--
 Author : Kumar Sindhurakshit
 Student Id: 200854641
 Date : 22-Feb-2022
 Version: 1.0
 Last Modified : 23-Feb-2022

This file defines launch parameters for advance robobotics week 5 test
-->
<launch>
   <group>
<!--
     <node name ="generator"  pkg="ar_week5_test" type="points_generator.py" output="screen" launch-prefix="xterm -e" />
     <node name ="coeff_compute_srv"  pkg="ar_week5_test" type="compute_cubic_coeffs.py" output="screen" launch-prefix="xterm -e" />
     <node name ="planner"  pkg="ar_week5_test" type="cubic_traj_planner.py" output="screen"  launch-prefix="xterm -e"/>
     <node name ="plotter"  pkg="ar_week5_test" type="plot_cubic_traj.py" output="screen" launch-prefix="xterm -e" />
     <node name ="rqt_plot"  pkg="rqt_plot" type="rqt_plot" args="/trjpos/x trjvel/y /trjacl/z" />
-->
     <node name ="generator"  pkg="ar_week5_test" type="points_generator.py" output="screen" />
     <node name ="coeff_compute_srv"  pkg="ar_week5_test" type="compute_cubic_coeffs.py" output="screen" />
     <node name ="planner"  pkg="ar_week5_test" type="cubic_traj_planner.py" output="screen" />
     <node name ="plotter"  pkg="ar_week5_test" type="plot_cubic_traj.py" output="screen" />
     <node name ="rqt_plot"  pkg="rqt_plot" type="rqt_plot" args="/trjpos/x trjvel/y /trjacl/z" />
     <node name ="rqt_graph"  pkg="rqt_graph" type="rqt_graph"  />


  </group>
</launch>
